.github/workflows/merge-node.js.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run formatting and linting across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Merge Build
+
+on:
+  pull_request:
+    branches: [ $default-branch ]
+    types: [ closed ]
+
+jobs:
+  merge_build:
+    # this job will only run if the PR has been merged
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'yarn'
+    - name: Install dependencies
+      run: yarn install --frozen-lockfile
+    - name: Build client package
+      run: CI='' yarn build --if-present
```

Comment:
I assume that the `CI` is used to fix some error?

Reply:
https://github.com/100devs/course-tracker/runs/4196173114?check_suite_focus=true

getting errors for warnings because CI=true when this was not set to empty string.

Reply:
This is the warnings we want to ensure aren't getting into the codebase. This should be a new issue.

Reply:
Removed the CI line from both workflow files (had removed from one only in prev commit to test)